docs/content/manual/release-specific/v1.1.0/rwx_feature.md

```diff
@@ -0,0 +1,320 @@
+10. Try to attach the PVC to another pod on another node, user should get `multi attach` error.
+
+# Restore an RWO backup into an RWW volume.
+1. Create a StatefulSet/Deployment with 1 pod.
+2. Attach a volume with RWX mode using longhorn class and selecting the option `read write once`.
+3. Write some data in the pod.
+4. Take a backup of the RWX volume.
+5. Restore from the backup.
+6. Verify the restored volume has `volume.spec.accessMode` as `rwo`.
+7. Create a PV and PVC with `accessMode` as `rww` for the restored volume.
+8. Verify the restored volume has `volume.spec.accessMode` as `rww` now.
+9. Attach a pod to the PVC and verify the data.
+10. Attach more pods to the PVC, verify the volume is accessible from multiple pods.
+
+# Create PV and PVC using Longhorn UI for RWX/RWO volume
+1. Create an RWX volume using Longhorn UI.
+2. Select the volume and create PV/PVC
+3. Verify the PV and PVC are created with `rwx` access mode.
+4. Create an RWO volume using Longhorn UI.
+5. Select the volume and create PV/PVC
+6. Verify the PV and PVC are created with `rwo` access mode.
+
+# Create RWX DR volume of a RWX volume in Longhorn.
+1. Create a StatefulSet/Deployment with 2 pods.
+2. Attach a volume with RWX mode using longhorn class and selecting the option `read write many`.
+3. Write some data in the pod.
+4. Take a backup of the volume.
+5. Create a DR volume of the backup by selecting `rwx` in access mode.
+6. Write more data in the pods and take more backups.
+7. Verify the DR volume is getting synced with latest backup.
+8. Activate the DR volume, attach it to multiple pods and verify the data.
+
+# Create RWO DR volume of a RWX volume in Longhorn.
+1. Create a StatefulSet/Deployment with 2 pods.
+2. Attach a volume with RWX mode using longhorn class and selecting the option `read write many`.
+3. Write some data in the pod.
+4. Take a backup of the volume.
+5. Create a DR volume of the backup by selecting `rwo` in access mode.
+6. Write more data in the pods and take more backups.
+7. Verify the DR volume is getting synced with latest backup.
+8. Activate the DR volume, attach it to a pod and verify the data.
+9. Try to attach it to multiple pods, it should show `multi attach error`.
+
+# Create RWX DR volume of a RWO volume in Longhorn.
+1. Create a StatefulSet/Deployment with 1 pod.
+2. Attach a volume with RWX mode using longhorn class and selecting the option `read write once`.
+3. Write some data in the pod.
+4. Take a backup of the volume.
+5. Create a DR volume of the backup by selecting `rwx` in access mode.
+6. Write more data in the pod and take more backups.
+7. Verify the DR volume is getting synced with latest backup.
+8. Activate the DR volume, attach it to multiple pods and verify the data.
+
+# Expand the RWX volume.
+1. Create a StatefulSet/Deployment with 2 pods.
+2. Attach a volume with RWX mode using longhorn class and selecting the option `read write many`.
+3. Write some data in the pod.
+4. Expand the volume.
```

Comment:
workloads need to be scaled down, since we only support offline expansion.

Reply:
Added the step for scaling down.